testplans/bitswap/_compositions/large.toml

```diff
@@ -0,0 +1,33 @@
+[metadata]
```

Comment:
At the moment the build configuration is coupled with the runner (i.e. you need all these configs under global.build_config in order for this plan to run correctly on k8s).

@coryschwartz Nit, but could you rename this file to `large-k8s.toml` so that it is clear that this composition is not going to work as is on `local:docker` for example.